### ISSUE TITLE
docker image for v1.0.1

### ISSUE THREAD

**Issue by kathryn-beal** [nunofonseca/irap#96]
Hi,
Could you let me know when a docker image for irap v1.0.1 will be available?
Best regards
Kathryn

**Comment by nunofonseca** [comment]
Hi Kathryn,
The docker images (Ubuntu and Fedora) for version 1.0.1 are now available in DockerHub.
Cheers.

**Comment by kathryn-beal** [comment]
Hi,
Thank you for doing this. I've tried pulling the latest docker image, however, I still have the same problem running it as I did for 1.0.0 (sh: 1: set: Illegal option -o pipefail). I can see this has been fixed in 1.0.1 so I'm a bit confused. If I look at the irap_version on github for v1.0.1 in, for example, aux/R/irap_utils.R:
irap_version<-"1.0.1"
but when I look in the image I downloaded I see:
irap_version<-"1.0.0"
I downloaded the image using singularity:
singularity pull docker://nunofonseca/irap_ubuntu:v1.0.1
Could I have done something wrong or is it possible that the 1.0.1 image was based from the 1.0.0 code base?
Best regards
Kathryn
